src/website/views/components/button/_examples.pug

```diff
@@ -519,19 +541,10 @@ h2 Examples
 .-portal--hide
   h3 Floating icon buttons
   .example.-mb--3
-    .chi-grid.-no-gutter
-      .chi-col.-w--12.-w-md--6
-        .-p--3.-d--flex(style="justify-content: center;")
-          each type, val in {Base:'', Primary:'primary', Dark:'dark'}
-            div.-pr--2
-              chi-button(color=type, type='float', alternative-text="Button action")
-                chi-icon(icon='atom')
-      .chi-col.-w--12.-w-md--6
-        .-p--3.-bg--black.-d--flex(style="justify-content: center;")
-          each type, val in {Secondary:'secondary', Light:'light'}
-            div.-pr--2
-              chi-button(color=type, type='float', alternative-text="Button action")
-                chi-icon(icon='atom')
+    .-p--3.-d--flex
+      div.-pr--2
+        chi-button(color='primary' type='float' size='xl' alternative-text='Chat with Support' data-tooltip='Chat with Support')
```

Comment:
Shouldn't floating icon btns in Lumen and Centurylink adhere to 48px instead of 56px too?

Reply:
@jllr Lumen and CenturyLink dot com use cases require larger sizes